### PR TITLE
Few Fixes

### PR DESCRIPTION
**Pull request by hostianer**
Chage IP Host Adress and added the correct .htaccess changes.

### CODE REVIEW

File: source/developers-guide/vagrant-phpstorm/index.md [hunk 1]
```diff
@@ -146,15 +146,30 @@ Input fields:
 After you get the `"BUILD SUCCESSFUL"` message you can run the full build command.
 
     $ ant build-unit
-  
+
+### Step 3
```
Comment (bcremer) [inline source/developers-guide/vagrant-phpstorm/index.md]: Is that still necessary? I've never used the `RewriteBase` for shopware installations on various setups. I pretty sure it works well on the vagrant machine without that step.

Comment (hostianer) [inline source/developers-guide/vagrant-phpstorm/index.md]: You`re right. A few minutes go and wasnt working. Now it works also if you leave it as it is :1

**Comment by bcremer** [comment]
Thank you @hostianer for your pull request.

Please create separate Pull Requests for each semantic change so we can review and merge them separately without blocking each other.

**Comment by hostianer** [comment]
@bcremer ok no problem will notice that for future pull requests.

**Pull request closed without merging**